api/models/graphite.go

```diff
@@ -11,6 +11,26 @@ import (
 	"gopkg.in/macaron.v1"
 )
 
+//go:generate msgp
+//msgp:ignore FromTo
+//msgp:ignore GraphiteFind
+//msgp:ignore GraphiteRender
+//msgp:ignore GraphiteTag
+//msgp:ignore GraphiteTagDetails
+//msgp:ignore GraphiteTagDetailsResp
+//msgp:ignore GraphiteTagDetailsValueResp
+//msgp:ignore GraphiteTagFindSeries
+//msgp:ignore GraphiteTagFindSeriesResp
+//msgp:ignore GraphiteTagResp
+//msgp:ignore GraphiteTags
+//msgp:ignore GraphiteTagsResp
+//msgp:ignore MetricNames
+//msgp:ignore MetricsDelete
+//msgp:ignore SeriesCompleter
+//msgp:ignore SeriesCompleterItem
+//msgp:ignore SeriesTree
+//msgp:ignore SeriesTreeItem
+
```

Comment:
you can just add the generate line to the types you want, then you don't neede all these ignores

Reply:
Are you sure about that? https://github.com/tinylib/msgp/wiki/Using-the-Code-Generator

Reply:
you're right. in fact having the generate lines for types as we do now in some other places is misleading because it actually works on the file level.